### DISCUSSION TITLE
Introductions | Please Introduce yourself before posting elsewhere.

### DISCUSSION THREAD

**Discussion by Philip-Clark** [club-40/odin-pantry#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as the main method of communication for the Club-40 Group Project. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
<hr>

**Comment by Philip-Clark** [comment]
Hello everyone, you know me as **BiFF**, feel free to address me as that or as Philip.
I'm very pleased with how many people have shown interest in this idea, let us all strive to make it a wonderful experience!

**Reply by ShivanshBhatnagar8** [reply]
Hi Vìçtør here, Just joined the club 40 and i m excited with the project

**Comment by Aureatus** [comment]
Hey folks, Aureatus here! Sure we'll all have a great time and learn a ton with this project ☺

**Comment by taladan** [comment]
Greetings and salutations 40'ers!  Taladan here, Hoping to be able to pitch in and help, as well as grow my own experience with this.

**Comment by CChiu28** [comment]
ScorpiusDX here. Excited to do this. 🤓

**Comment by JoshDevHub** [comment]
Hello, I'm Josh (josh#0729 on discord).

I'm not 100% sure how much I'll be able to contribute (or if I'll be able to contribute) in terms of writing code. But I figure I can give some advice on things to hopefully get the ball rolling.

**Reply by Philip-Clark** [reply]
If you are willing to help with maintenance, give advice, or lead, feel free to say so here: https://github.com/Philip-Clark/TOP-Club-40-Group-Project/discussions/7

**Comment by Ananth-Adhikarla** [comment]
Hello, I am Ananth or you might know me as Promise.resolve("Success") and I am excited to be a part of this.

**Comment by Cedrus32** [comment]
Hi everyone, it's Cedrus32. Excited to see this project takes shape!

**Comment by Baguirre03** [comment]
Hey everyone! Its rayray (Ben) here, not sure if I will be able to do the most but excited to see where this goes!

**Comment by fcasibu** [comment]
Hey, I'm `nevz#9704` glad to see everyone joining. I also am not sure how much I can contribute but I'll see what I can do

**Comment by RyanMcEntire** [comment]
Hey, `Clowdy#1000` here. Looking forward to helping however I can. I submitted an idea in the google form, and i'm looking forward to seeing other peoples ideas.

excited to see how this plays out!

**Comment by justEstif** [comment]
Hello, I am Estifanos (`_steve#4245`). Happy to be here and contribute.

**Comment by 01zulfi** [comment]
Hey hey! xari from Discord here. Super excited where this is headed 🚀

**Comment by ManorSailor** [comment]
Hello, it's me, `Sars#8753` - if you didn't already guess with my weird username xD. Anyways, really looking forward to what we, as a community, can create. 
PS: I wasn't aware of this discussion thread before, otherwise, I would have introduced myself earlier.

**Comment by crespire** [comment]
Happy to help with the admin side of things, but probably won't be contributing code!

**Reply by Philip-Clark** [reply]
If you are willing to help with maintenance, give advice, or lead, feel free to say so here: https://github.com/Philip-Clark/TOP-Club-40-Group-Project/discussions/7

Thanks for dropping by! 🎉

**Comment by amcmillion3** [comment]
Hey, I'm Pandawg.

**Comment by mmxz-mmxz** [comment]
Hello - Ombrah here 👋

**Comment by Zadag** [comment]
Hello all, KB#1707 from the TOP server.  I'm most comfortable with React, JS (and a bit of TS probably wouldn't be too difficult to pick up more), Tailwind, and SQL.   I plan on writing a lot of code but I'm not confident in my ability to manage a project of this size.  Looking forward to see where this goes!

**Comment by rcamach7** [comment]
HI, I am Ricardo, AKA Mireles#1596 on Discord. I am interested in making contributions and helping in whichever way I can!

**Comment by JustWaveThings** [comment]
Hey I am .wav or Wave#4186  from Discord. I am just at the beginning of React, so I'm not sure how much I can help, but I'd like to see If I can make a small contribution.

**Comment by Lofty-Brambles** [comment]
Hey there! I'm Lofty Brambles, over from discord. I'm open to helping out with reviews and code as necessary!

**Comment by Mogoatlhe** [comment]
Hello, I'm `mogoatlhe#9903` from the TOP discord server. I've been following the discussion on discord and would like to express my interest in taking part of this. it sounds amazing, great job so far with putting everything together. :muscle: 

Edit: my last project was the `Inventory App` in the node section.

**Comment by royojeda** [comment]
Hi, I'm Dennis (`Dennis#2289` on Discord). Looking forward to what we can make of this!

**Comment by FyendFyre** [comment]
Hello everyone! My name's Saad (`wyndfyre#7055` on discord). You call me Saad or Wynd or WyndFyre. whatever works for you. I'm excited and ready to see how this turns out and what experience we all can gain from this!

**Comment by MarchandMD** [comment]
Hey gang,

I've been keeping one eye on this project take shape, figured I should introduce myself in case I want to contribute.

Name's Michael IRL or ~~Csizzle~~ `milo minderbender` on the Discord (Catch-22 reference). Happy to be here. 🎉

**Reply by MarchandMD** [reply]
@Philip-Clark thanks! glad to be able to pitch in. Let's keep the ball rolling!

**Reply by Philip-Clark** [reply]
Sorry that you only just now received an invite. I thought for sure I sent invites to all of the original group 😓
Glad to have you!

**Comment by justindanielclark** [comment]
Hi All,

Matriarchy on the discord.

**Comment by Zekumoru** [comment]
Greetings everyone! :smile: 
My name's the same as on Discord. If there's something I could help with, I'd be happy to! Such a wonderful idea to bring us all together in this amazing project! :100:

**Reply by Zhiul** [reply]
Hi! Zhiul#5852 here, I want to follow this collaborative project and try to make some contributions. 😃

**Comment by fiufiu0** [comment]
Hi! I'm `fiufiu` on Discord server. If I can help with anything, I'll be glad. 💪 Nice project initiative. 👌

**Comment by JaeGif** [comment]
Hey guys. Im Jae, ItsJae on the discord. This is some sick stuff you’ve got going on. Count me in! 
Edit: I've finished the curriculum Node.js.

**Comment by Asartea** [comment]
'Ello, I'm Asarta (console.error("Asarta”)) on Discord

**Comment by Kingbot13** [comment]
Hi everyone, I'm quarterpotato (real name is Dylan) on Discord. I just recently joined club 40 and I have completed the entire JavaScript path in TOP. I would love to be a part of the group project!

**Comment by impronen** [comment]
Hi y'all 

I'm antero from the Discord. Want to help out with some small contributions to this project.

**Comment by MooseCowBear** [comment]
Hey everyone. I'm Alisa. I'm a Rails person mostly, although I wouldn't mind the occasional opportunity to learn more TypeScript.

**Comment by damon314159** [comment]
Hello :wave: I'd love to get stuck in \:)
My discord name is the same

**Comment by LexiphanicStroon** [comment]
Hello everyone, I'm an application developer for an ISP and would love to help out on this project in any way I can!
I work largely with TailwindCSS, vanilla JS, React, PHP, and am in charge of UI and UX design for the development department at my job. 

Discord tag is LexiphanicStroon

**Comment by isaiahaiasi** [comment]
Hi y'all! I'm Isaiah (`isaiahaiasi` on discord). I've gone through the Node path, and I've got a fair bit of TypeScript experience. I'd love to pitch in with this project! 😁
